examples/cluster_scoped_custom_object.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Uses a Custom Resource Definition (CRD) to create a custom object, in this case
```

Comment:
usually it is called 'custom resource' (CR) instead of 'custom object'

Reply:
corrected `custom object` to `Custom Resource (CR)`. Thanks for pointing.